composer.lock

```diff
@@ -732,46 +732,6 @@
             ],
             "time": "2016-09-16T12:50:15+00:00"
         },
-        {
-            "name": "damianopetrungaro/cachet-sdk",
```

Comment:
Why have you changed something in here?

Reply:
I thought i would need this package but i didn't so i removed it.